inst/qml/qml_components/BayesianMetaAnalysisPriors.qml

```diff
@@ -0,0 +1,241 @@
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public
+// License along with this program.  If not, see
+// <http://www.gnu.org/licenses/>.
+//
+import QtQuick 2.8
+import QtQuick.Layouts 1.3
+import JASP.Controls 1.0
+import JASP.Widgets 1.0
+
+Section
+{
+	title: 		qsTr("Priors")
+	columns:	1
+
+	property string modelTypeValue:			"BMA"
+	property string modelDirectionValue:	"allPos"
+
+	Group
+	{
+		columns: 2
+
+		RadioButtonGroup
+		{
+			title: 	qsTr("Effect size")
+			name: 	"priorES"
+
+			RadioButton
+			{
+				id: 				cauchyInformative
+				label: 				qsTr("Cauchy")
+				name: 				"cauchy"
+				checked: 			true
+				childrenOnSameRow: 	true
+
+				DoubleField
+				{
+					label: 			qsTr("location:")
+					name: 			"informativeCauchyLocation"
+					visible: 		cauchyInformative.checked
+					defaultValue: 	0
+					negativeValues: true
+				}
+
+				DoubleField
+				{
+					label: 			qsTr("scale:");
+					name: 			"informativeCauchyScale"
+					visible: 		cauchyInformative.checked
+					defaultValue: 	0.707
+					fieldWidth: 	50
+				}
+
+
+			}
+
+			RadioButton
+			{
+				id: 				normalInformative
+				label: 				qsTr("Normal")
+				name: 				"normal"
+				childrenOnSameRow:	true
+
+				DoubleField
+				{
+					label: 			qsTr("mean:")
+					name: 			"informativeNormalMean"
+					visible: 		normalInformative.checked
+					defaultValue: 	0
+					negativeValues: true
+				}
+
+				DoubleField
+				{
+					label: 			qsTr("std:")
+					name: 			"informativeNormalStd"
+					visible: 		normalInformative.checked
+					defaultValue: 	0.707
+					fieldWidth: 	50
+				}
+
+
+			}
+
+			RadioButton
+			{
+				id: 				tInformative
+				label: 				qsTr("t")
+				name: 				"t"
+				childrenOnSameRow: 	true
+
+				DoubleField
+				{
+					label: 			qsTr("location:")
+					name: 			"informativeTLocation"
+					visible: 		tInformative.checked
+					defaultValue: 	0
+					negativeValues: true
+				}
+
+				DoubleField
+				{
+					label: 			qsTr("scale:")
+					name: 			"informativeTScale"
+					visible: 		tInformative.checked
+					defaultValue: 	0.707
+					fieldWidth: 	50
+				}
+
+				IntegerField
+				{
+					label: 			qsTr("df:");
+					name: 			"informativeTDf";
+					visible: 		tInformative.checked;
+					min:			1
+					defaultValue: 	1
+				}
+
+
+			}
+		}
+		Group
+		{
+			title: qsTr("Truncation")
+			CheckBox
+			{
+				name: 				"checkLowerPrior"
+				childrenOnSameRow: 	true
+				checked: 			modelTypeValue == "CRE" && modelDirectionValue == "allPos"
```

Comment:
@boutinb isn't it recommended to always use `===` rather than `==`? To avoid accidental coercions?

Reply:
fixed